pmd-apex/src/main/resources/category/apex/design.xml

```diff
@@ -36,6 +36,64 @@ public class Foo {
         </example>
     </rule>
 
+    <rule name="AvoidUnusedMethod"
```

Comment:
Maybe the last thing I would change is to name this rule `UnusedMethod`. I'm not a fan of rule names which start with a verb... I think the passive is more neutral. Like "hey, I found an unused method (but of course you already know that you should avoid them, so I'm not commanding you)"

Besides, the corresponding Java rules are also named without `Avoid`, and consistency across languages is desirable I think

Reply:
@nawforce did you see this?

Reply:
Yep thx, just pushed an update that changes name.